test/externalTests/common.sh

```diff
@@ -258,7 +265,7 @@ function truffle_run_test
         clean
         force_solc_settings "$CONFIG" "$optimize" "istanbul"
         # Force abi coder v2 in the last step. Has to be the last because code is modified.
-        if [ "$FORCE_ABIv2" = true ]; then
+        if [ "$force_abi_v2_flag" = force_abi_v2 ]; then
```

Comment:
Could this be retained as a boolean?

Reply:
I specifically did not want this to be a boolean to make the call more self-explanatory. I prefer this:

```bash
truffle_run_test "$SOLJSON" compile_fn test_fn no_force_abi_v2
```

to this:
```bash
truffle_run_test "$SOLJSON" compile_fn test_fn false
```

I also try to avoid using `true` and `false` as booleans in Bash when I have a good alternative because it can easily bite you in some contexts. Especially the fact that `if [ $var = true ]` and `if [ $var = false ]` will just silently do the wrong thing when `var` has a value other than `true` or `false`.

Admittedly, `no_force_abi_v2` looks too much like a function name in this context. I'll rename it to something else.

By the way, no test is currently using this flag so I could also just remove it.